src/zod-extensions.ts

```diff
@@ -5,8 +5,10 @@ import { isZodType } from './lib/zod-is-type';
 export interface ZodOpenAPIMetadata<T = any> extends SchemaObject {
   refId?: string;
   extendedFrom?: string;
-  param?: Partial<ParameterObject> & { example?: T };
-  example?: T;
+  param?: Partial<ParameterObject> & {
+    example?: T extends Date ? string : T;
```

Comment:
@heivo can you please pull the latest master. Since in [this PR](https://github.com/asteasolutions/zod-to-openapi/pull/53/files) the same typing has been used for `examples` and `default`. You can also create a reusable type to remove the duplication.